source/api/presentation/3.0/index.md

```diff
@@ -240,6 +240,19 @@ The value _MUST_ be a JSON array, with each item in the array being a JSON objec
 {"thumbnail": [{"id": "https://example.org/img/thumb.jpg", "type": "Image"}]}
 ```
 
+##### posterCanvas
+A single Canvas providing content that represents this resource, such as image to show while a duration-only Canvas is playing audio, or images, text and sound shown for video content before the user initiates playback. Whereas a client might use `thumbnail` for navigation or summary views of multiple resources, a client can choose to show content from `posterCanvas` as part of the presentation of this resource without requiring the content of `posterCanvas` to be part of this resource. The poster Canvas can have different dimensions to the Canvas(es) of this resource.
```

Comment:
"represents" seems too restrictive -- an ad for a product that's displayed during the audio of a podcast manifest ... does the ad represent the manifest? 
represent -> is associated with ?

Reply:
Is there a use case for canvases without duration to have a posterCanvas?  If a posterCanvas is associated with a Manifest that contains Canvases without duration (and some that do) ... what does the client do? Flick between the h/w canvas content and the poster content?

Reply:
Avoided notion of representation - instead, _associated with but not part of_

How much should the spec say about sensible use of posterCanvas? I think the spec should be silent on complexities like "Flick between the h/w canvas content and the poster content" with just the general indication that the client _MAY_ show the most appropriate content for the user's context. In practice I would hope that use of posterCanvas is limited to Manifest and perhaps Range (segments or episodes maybe).